app/static/js/NeighbourPlotCanvas.js

```diff
@@ -274,7 +342,7 @@ NeighbourPlotCanvas.prototype._createNavigateBehaviour = function() {
     return d3.behavior.zoom()
         .x(this.xScale)
         .y(this.yScale)
-        .scaleExtent(config.scaleExtent)
+        .scaleExtent(config.zoomExtent)
```

Comment:
No, I renamed it from scaleExtent to zoomExtent in the config. Less
ambiguity!

On Tue, Feb 16, 2016 at 7:07 PM, Juan Nunez-Iglesias <
notifications@github.com> wrote:

> In app/static/js/NeighbourPlotCanvas.js
> <https://github.com/microscopium/microscopium-ui/pull/78#discussion_r52978236>
> :
>
> > @@ -274,7 +342,7 @@ NeighbourPlotCanvas.prototype._createNavigateBehaviour = function() {
> >      return d3.behavior.zoom()
> >          .x(this.xScale)
> >          .y(this.yScale)
> > -        .scaleExtent(config.scaleExtent)
> > +        .scaleExtent(config.zoomExtent)
>
> Is this an error from a previous PR? If so, is it feasible to write a test
> that catches it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/microscopium/microscopium-ui/pull/78/files#r52978236>.
>